loader/internal/loader/loader.go

```diff
@@ -212,6 +254,48 @@ func (l *L) loadBaseModel(ctx context.Context, modelID string) error {
 	return nil
 }
 
+func (l *L) loadModel(ctx context.Context, model config.ModelConfig) error {
+	if err := l.loadBaseModel(ctx, model.BaseModel); err != nil {
+		return err
+	}
+
+	// HuggingFace uses '/" as a separator, but Ollama does not accept. Use '-' instead for now.
+	// TODO(kenji): Revisit this.
+	convertedModelID := strings.ReplaceAll(model.Model, "/", "-")
+
+	// First check if the model exists in the database.
+	ctx = auth.AppendWorkerAuthorization(ctx)
+	_, err := l.modelClient.GetModelPath(ctx, &mv1.GetModelPathRequest{Id: convertedModelID})
+	if err == nil {
+		l.log.Info("Model", convertedModelID, " exists. Do nothing.")
```

Comment:
Let's follow the style of the structured logging. 

```suggestion
		l.log.Info("Already model exists", "modelID", convertedModelID)
```